### DISCUSSION TITLE
Remove-Variable : Cannot bind argument to parameter 'Name' because it is null.

### DISCUSSION THREAD

**Discussion by nirajacharyaa** [JanDeDobbeleer/oh-my-posh#1153] [Troubleshoot] [answered]
![image](https://user-images.githubusercontent.com/54741119/139797740-954b7417-4e43-4025-8966-379e8ab1948a.png)

How do I solve these errors, I cannot find any config file having a line number referred by the error.

**Comment by JanDeDobbeleer** [comment, marked as answer]
Please update to 5.16.6

**Comment by nirajacharyaa** [comment]
Thanks, I saw the closed PR which fixed this problem just after opening this discussion.
